rllib/algorithms/apex_dqn/apex_dqn.py

```diff
@@ -634,28 +627,34 @@ def update_target_networks(self, num_new_trained_samples) -> None:
                 else NUM_ENV_STEPS_TRAINED
             ]
 
-    @override(Algorithm)
-    def on_worker_failures(
-        self, removed_workers: List[ActorHandle], new_workers: List[ActorHandle]
-    ):
-        """Handle the failures of remote sampling workers
+    def _get_shard0_replay_stats(self) -> Dict[str, Any]:
+        """Get the replay stats from the replay actor shard 0.
```

Comment:
Can we add a description on what this means?
Like: get list of healthy workers, pick the first one from that list.
Also: Why would we not want to collect all workers' stats? For performance reasons?

Reply:
done.
I have no idea why we did this man. I think we should collect replay stats from all shards.

Reply:
Hmm, probably performance. No idea other than that either. :)